pkg/config/config_template.yaml

```diff
@@ -51,6 +51,12 @@ api_key:
 # Warning: percentiles must be specified as yaml strings
 #
 # histogram_percentiles: ["0.95"]
+#
+# Copy histgram values to distributions for true global distributions (in beta)
```

Comment:
Would this impact billing ? I'm thinking to the number of custom metrics we bill. If so we should add a comment about it.

Reply:
Not while we're in beta, but yes eventually. Will add something about it.

Reply:
typo: `histogram`